src/main.cpp

```diff
@@ -646,11 +646,19 @@ unsigned int LimitOrphanTxSize(unsigned int nMaxOrphans) EXCLUSIVE_LOCKS_REQUIRE
 
 
 
-bool IsStandardTx(const CTransaction& tx, string& reason)
+bool IsStandardTx(const CTransaction& tx, string& reason, uint32_t consensusBranchId)
```

Comment:
I think this can be refactor later on as currently quite a few places pass in the reason parameter.  If we changed it now, a whole bunch of code elsewhere would change and it's probably better to keep the Overwinter changes as small as possible for now